src/docfx/build/dependency/DependencyResolver.cs

```diff
@@ -159,13 +159,13 @@ public string GetRelativeUrl(Document fileRelativeTo, Document file)
                     }
                     var selfUrl = Document.PathToRelativeUrl(
                         Path.GetFileName(file.SitePath), file.ContentType, file.Schema, file.Docset.Config.Output.Json);
-                    return (error, selfUrl + query + fragment, fragment, hrefType, null);
```

Comment:
should `hrefType` be `HrefType.SelfBookmark` in the first place? I mean, the output of https://github.com/dotnet/docfx/blob/c55b52837b77150f9c37020323231e9ca3129321/src/docfx/build/dependency/DependencyResolver.cs#L196

Reply:
it should be, but to keep the behavior same with v2([Self](a.md#test) => <a href=“a#test”>), we cannot generate the right hrefType  in the first place